### PR TITLE
fix: fix opentelemetry-operator.manager.env value

### PR DESCRIPTION
**Pull request by sumo-drosiek**
Fix default value to validation error:

```
E           Error: INSTALLATION FAILED: values don't meet the specifications of the schema(s) in the following chart(s):
E           opentelemetry-operator:
E           - manager: env is required
```

### Checklist

- [ ] Changelog updated or skip changelog label added
- [ ] Documentation updated
- [ ] Template tests added for new features
- [ ] Integration tests added or modified for major features

### CODE REVIEW

**Comment by swiatekm-sumo** [comment]
Where did you encounter this error and under what circumstances? The current value has integration tests passing.

**Comment by sumo-drosiek** [comment]
In our e2e tests for 
```
helm install ... sumo-repo/sumologic  --set=image.tag=3.14.0-46-g543a433e2f -f values.yaml --set sumologic.endpoint= --set sumologic.accessId= --set sumologic.accessKey= --set sumologic.collectorName= --set sumologic.clusterName=  --version 3.14.0-46-g543a433e2f --devel
```

**Comment by swiatekm-sumo** [comment]
Are you sure this change actually does anything. I think `env:` and `env: {}` are equivalent in yaml.

**Comment by sumo-drosiek** [comment]
```
Error: INSTALLATION FAILED: values don't meet the specifications of the schema(s) in the following chart(s):
opentelemetry-operator:
- manager: env is required

➜  tmp git:(drosiek-fix-schema) cat values.yaml 
opentelemetry-operator:
  manager:
    env: nul
```

**Comment by sumo-drosiek** [comment]
Created an upstream issue

**Review by mat-rumian — APPROVED** [review]

**Comment by swiatekm-sumo** [comment]
@sumo-drosiek github upgraded Helm to 3.13 in their runners and now this causes an error. So we should probably merge this change.

**Comment by sumo-drosiek** [comment]
Default is different I will update the PR

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -2436,7 +2436,7 @@ opentelemetry-operator:
     collectorImage:
       repository: "public.ecr.aws/sumologic/sumologic-otel-collector"
       tag: "0.85.0-sumo-0"
-    env:
+    env: {}
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/values.yaml:2439]:
```suggestion
    env:
      ENABLE_WEBHOOKS: "true"
```

File: deploy/helm/sumologic/README.md [hunk 2]
```diff
@@ -275,7 +275,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
...
-| `opentelemetry-operator.manager.env`                                                  | Additional environment variables for opentelemetry-operator helm chart.                                                                                                                                                                                                                                    | `Nil`                                                                                                                                                                                |
+| `opentelemetry-operator.manager.env`                                                  | Additional environment variables for opentelemetry-operator helm chart.                                                                                                                                                                                                                                    | `{}`                                                                                                                                                                                 |
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/README.md:278]:
```suggestion
| `opentelemetry-operator.manager.env`                                                  | Additional environment variables for opentelemetry-operator helm chart.                                                                                                                                                                                                                                    | `{"ENABLE_WEBHOOKS": "true"}`                                                                                                                                                        |
```

**Comment by sumo-drosiek** [comment]
@swiatekm-sumo @mat-rumian TBH not sure which version is better, duplication of default values or ignoring them

**Pull request merged**